File: medplum-java/medplum-model/src/main/java/com/fhir/QuestionnaireResponse_Item.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableQuestionnaireResponse_Item.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface QuestionnaireResponse_Item {
  Optional<List<QuestionnaireResponse_Item>> item();

  Optional<List<Extension>> modifierExtension();

  Optional<String> id();

  Optional<List<Extension>> extension();

  Optional<uri> definition();

  Optional<List<QuestionnaireResponse_Answer>> answer();

  Optional<String> linkId();

  Optional<String> text();

  static ImmutableQuestionnaireResponse_Item.Builder builder() {
    return ImmutableQuestionnaireResponse_Item.builder();
  }
}

File: medplum-typescript/src/model/fhir/QuestionnaireResponse_Item.ts
import { Extension } from "./Extension";
import { QuestionnaireResponse_Answer } from "./QuestionnaireResponse_Answer";
import { QuestionnaireResponse_Item } from "./QuestionnaireResponse_Item";
import { uri } from "./uri";

export interface QuestionnaireResponse_Item {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    linkId: string | null | undefined;
    definition: uri | null | undefined;
    text: string | null | undefined;
    answer: QuestionnaireResponse_Answer[] | null | undefined;
    item: QuestionnaireResponse_Item[] | null | undefined;
}
